vinhloc30796/creativecontact-app
# feat(artwork): add video support to ArtworkCard component

## Description

This PR adds video support to the ArtworkCard component, allowing the display of video thumbnails for artwork assets. It also updates the data structure to include asset type information.

## Key Changes

1. Updated ArtworkCard component to support video thumbnails
2. Modified ArtworkWithAssetsThumbnailCredits type to include assetType
3. Updated EventPage to pass assetType to ArtworkCard
4. Expanded allowed MIME types in Supabase storage configuration
5. Added seed data for Supabase Storage to test video

## Breaking Changes

- The `ArtworkWithAssetsThumbnailCredits` type now includes `assetType` in the `thumbnail` object, which may require updates in components using this type.

## Related Issues

- Closes #WEB-127

## Testing Instructions

1. Navigate to an event page with artwork that includes video assets
2. Verify that video thumbnails are displayed correctly in the ArtworkCard component
3. Click on a video thumbnail to ensure it plays properly
4. Check that existing image thumbnails still work as expected

## Additional Notes

- This change requires updating the Supabase storage configuration to allow video file types. Make sure to apply the changes in the `supabase/config.toml` file to your Supabase project settings.
- The seed data has been updated to include a sample video asset for testing purposes.
